scripts/update-clang-version.sh

```diff
@@ -1,24 +1,22 @@
 #!/bin/bash
 
+set -exuo pipefail
```

Comment:
There are even more flags (see "man set") I do not know why I forgot to use that.